ui/apps/dashboard/src/components/Navigation/old/AppNavigation.tsx

```diff
@@ -1,23 +1,24 @@
-'use client';
```

Comment:
The next few nav components have been altered to permit server fetch and render..